### PR TITLE
Update English title page

### PR DESCRIPTION
**Pull request by mukrop**
Preview build: https://en-facts-on-climate--preview-update-titlepage-cj7xdqcb.web.app/
(I don't know why, but my Chrome flags it as a dangerous site...)

Context: I managed to get both the Czech and English website as one of the featured resources in Red Hat's "Day of Learning" on sustainability for 2022-12-09. Therefore, I'm updating the English webpage to be more representative.

This PR updated the title page of the English website in the following ways:
* The "newest" section is replaced by "featured" section.
  * Rationale: We don't publish much here and the latest objects were on Czech energy transformation, much less relevant for an average reader.
* The top part of the page has updated text based on the one on the Czech website.
  * Rationale: The old was very old, no longer valid.
* Controversial: There is a new disclaimer box under the featured infographics.
  * I wanted to explain it's a smaller cousin of the Czech project with far less content.
* The bottom of the title page is updated to reflect the current Czech page.
  * Rationale: Better explains the project than the old texts. Presents awards and institutions that use us.
* The "About" page has the section on support and partnerships (copied from CZ web).
  * Rationale: This seemed necessary to be transparent on financing and similar.
* Note: Some logos have been exchanged for their English version. Images has been cleaned in the same folder locations as in the Czech website.

@mgrabovsky or @jankrcal, can you please review this in the upcoming days (no need for tomorrow, but would be great to have it deployed in a week). Thanks!

### CODE REVIEW

File: collections/_pages/index.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ sitemap: true
 <div class="section intro pb-4">
     <div class="container">
         <h1 class="display-1" id="home">Facts on Climate<br>Change</h1>
-        <p>We collect various climate change data provided by Czech and international research and monitoring institutions (CHMI, NASA, Eurostat, etc.) and present this data in our graphs and infographics for <a href="/how-to-use" title="How to use our materials">further use</a>.<br/>
+        <span class="tagline">Publicly available <span class='nobr'>infographics, explainers and data</span></span>
+        <p>We are a team of independent analysts and experts, committed to cultivating the debate on climate change to be factual, constructive and based on science and verified data.<br/>
```
Comment (mgrabovsky) [inline collections/_pages/index.md]: The line break is unnecessary here.
```diff
-        <p>We are a team of independent analysts and experts, committed to cultivating the debate on climate change to be factual, constructive and based on science and verified data.<br/>
+        <p>We are a team of independent analysts and experts, committed to cultivating the debate on climate change to be factual, constructive and based on science and verified data.
```

**Review by mgrabovsky — APPROVED** [review]
Looks good to me. Thanks for the update. Feel free to merge as convenient.

File: collections/_pages/about.html [hunk 2]
```diff
@@ -5,48 +5,63 @@
...
+                    {%- if item.twitter %}<a class="no-ext-link-icon" href="{{ item.twitter }}"><span class="fab fa-fw fa-twitter"></span></a>{% endif %}
+                </h5>
+                <h6 class="card-subtitle mb-3 text-muted text-center">{{ item.position }}</h6>
+                <div class="card-text mb-3 text-center">{{ item.summary | markdownify }}</div>
+                {%- if item.email %}
+                <div class="mt-auto text-center">
+                    <a class="align-self-end" href="mailto:{{ item.email }}">{{ item.email }}</a>
                 </div>
-                {%- endfor %}
+                {% endif -%}
```
Comment (jankrcal) [inline collections/_pages/about.html]: did you mean {%- endif %}?
(not sure what a trailing -% does, if it's equivalent to a leading %-)

File: collections/_pages/index.md [hunk 3]
```diff
@@ -7,16 +7,30 @@ sitemap: true
 <div class="section intro pb-4">
     <div class="container">
         <h1 class="display-1" id="home">Facts on Climate<br>Change</h1>
-        <p>We collect various climate change data provided by Czech and international research and monitoring institutions (CHMI, NASA, Eurostat, etc.) and present this data in our graphs and infographics for <a href="/how-to-use" title="How to use our materials">further use</a>.<br/>
+        <span class="tagline">Publicly available <span class='nobr'>infographics, explainers and data</span></span>
```
Comment (jankrcal) [inline collections/_pages/index.md]: I'd simplify to "infographics and explainers". "Data" is quite unclear in this context, imo.

File: collections/_pages/index.md [hunk 4]
```diff
@@ -7,16 +7,30 @@ sitemap: true
...
+<p class="lead mb-0">New to the topic? Want to learn about climate change, its relationship to the level of emissions, temperature change and future predictions? Start with the infographics below.</p>
+{% assign featured_slugs = "climate-change-scheme, concentration-warming-relationship, map-temperature-change, emission-pathways-paris, fossil-fuels-emissions, tipping-points-1" | split: ", " %}
+{%- include preview-blocks-expandable.html slugs=featured_slugs rows=2 %}
+<div class="alert alert-info data-staleness" role="alert">
+    <h5 class="alert-heading"><i class="fas fa-lightbulb"></i> Work in progress </h5>
+    <p>This website is the initial English version of a successful Czech project <a href="https://faktaoklimatu.cz">Fakta o klimatu</a>. At the moment, only a fraction of the content is available in English. If you like what you see, you can support further work via the <a href="{{ site.benevity }}">Benevity Cause</a> via your employer CSR program or directly using the <a href="{{ site.fundraising }}">Czech donation page</a>.</p>
+</div>
+</div></div>
+
+{%- comment -%}
```
Comment (jankrcal) [inline collections/_pages/index.md]: I'd just delete that fragment but I don't feel strongly.

File: collections/_pages/index.md [hunk 5]
```diff
@@ -57,30 +72,75 @@ Climate change is a complex of interrelated phenomena. That is why the data that
...
-{:.lead}
-We are a team of professionals with various skills and expertise – science, IT, communication and teaching – and we communicate with journalists, politicians, stakeholders as well as activists. We are an independent think tank and our budget is based on donations and grants.
+**We communicate to key audiences**. We create data visualizations, articles, podcasts or lectures for the general public and clients from both the private and public sectors.
+
+</div>
+<div class="col-12 col-md-6 col-lg-5 pt-4" markdown="1">
+### We strive for
+
+{:.arrow}
+* **Clear and publicly accessible information** on climate change and possible pathways to a zero-emission society
```
Comment (jankrcal) [inline collections/_pages/index.md]: zero-emission -> net-zero?

File: collections/_pages/index.md [hunk 6]
```diff
@@ -57,30 +72,75 @@ Climate change is a complex of interrelated phenomena. That is why the data that
...
+    <div class="col-12 col-md-6 col-lg-4 p-3 p-md-4 price">
+        <div class="price-1"></div>
+        <div>
+            <strong>Award for the communication of climate change</strong> (2020) from the Czech Learned Society and United Nations Information Centre in Prague
+        </div>
+    </div>
+    <div class="col-12 col-md-6 col-lg-4 p-3 p-md-4 price">
+        <div class="price-2"></div>
+        <div>
+            First price in the <strong>Climate Challenge</strong> accelerator (2020)
```
Comment (jankrcal) [inline collections/_pages/index.md]: ...in the Czech <strong>Climate Challenge</strong>...?

File: collections/_pages/index.md [hunk 7]
```diff
@@ -57,30 +72,75 @@ Climate change is a complex of interrelated phenomena. That is why the data that
...
 
-{:.lead}
-We would like to contribute to a more evidence-based Czech public debate about climate change.
-We create easy-to-understand visual and text materials and communicate the latest research findings to support key decisions in the Czech Republic that are related to climate change and a transition to a net zero economy.
+<div class="row about-us lead mb-5 justify-content-between">
+<div class="col-12 col-md-6 pt-2 pt-md-4" markdown="1">
 
-{:.lead}
-The debate about climate change is complicated and full of emotions, half-truths and myths. We want to highlight the most important issues and build bridges between scientists and all people affected by climate change. Which means all people.
+**We do research and analytical work**. We browse the data and look for aspects relevant to a deeper understanding of climate change, its impacts and possible solutions. We accentuate the regional matters of Czechia and Europe but do not omit global views.
```
Comment (jankrcal) [inline collections/_pages/index.md]: alternative (not feeling strongly)

We sift through the data, searching for bits that help a deeper understanding of climate change, its impacts and possible solutions.

File: collections/_pages/about.html [hunk 8]
```diff
@@ -5,48 +5,63 @@
...
+    {%- endfor %}
+</div>
 {%- endfor %}
+
+</div></div>
+<div class="section"><div class="container">
+
+<h2>Support and partnerships</h2>
+<div class="lead narrow-text">
+    <p>Support and cooperation with other partners is crucial to the success of our work. In order to devote the necessary time to creating quality research, infographics and explainers, we need sufficient funding. That enables us to fully commit to making our outputs to lead to constructive discussion on climate change and related societal transformation and to data-driven decisions.</p>
```
Comment (jankrcal) [inline collections/_pages/about.html]: The last sentence sounds cumbersome to me. I'd deviate from the Czech text and write:

That also helps us communicate our findings and thus contribute to a more constructive debate on climate change and to data-driven decisions towards a net-zero society.

**Review by jankrcal — APPROVED** [review]
Looks good to me. Some pieces of English would benefit from Jirka's love but it meets to bar to me. Below, I propose a few edits, take a liberty to ignore them or to incorporate them...

File: collections/_pages/about.html (hunk 2 above)
Comment (mukrop) [reply collections/_pages/about.html]: The [documentation](https://shopify.github.io/liquid/basics/whitespace/) is not very specific there :D. I did not write this line, just move it. Nevertheless, I moved it to unify.

**Comment by mukrop** [comment]
Thanks to both for the comments, I've incorporated all of them.
In general, I do agree the language could be better but I want to try using the website (the RH learning day) and consider imperfect texts much better than none (if they are above the acceptance bar).

**Pull request merged**